### PR TITLE
Dripline Section

### PR DESCRIPTION
**Pull request by garrying**
### topline text

> A dripline signifies the area under the outermost branches of a tree.
> It’s this [interface](https://permacultureprinciples.com/principles/_11/) between worlds where interesting and productive things happen.

## To Do

- [x] reduce page image to 55% width https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779450411
- [x] add topline text to post layout https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779450411
- [x] add topline text to index https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779450411
- [x] move footer into post layout https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779450411
- [x] fix dweb deploy script https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779453762
- [x] fix asset file refs (start with dot, or no slash) https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779453762 Resolutions: https://github.com/jekyll/jekyll/issues/6360
    - https://ipfs.distributed.press/ipns/hypha.coop/
- [x] ~~design contrast fix https://github.com/hyphacoop/hypha.coop/pull/21#issuecomment-779453762~~ `reticket`

### CODE REVIEW

File: _posts/2021-01-23-how-we-co-operate.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+We’re excited to hear from you!
+Reach out if there are topics you want to discuss or you think we should cover in future posts.
+Our email is: [dripline@hypha.coop][email]
+
+   [email]: mailto:dripline@hypha.coop
+
+---
+
+This site is published using [Distributed Press][], an initiative we host.
```
Comment (uditvira) [inline _posts/2021-01-23-how-we-co-operate.md]: Hypha.coop is published using Distributed Press, not just Dripline. Can this be in the footer across the board?

File: _posts/2021-01-23-how-we-co-operate.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+We’re excited to stand with others working to that end.
+
+   [just recovery for all]: https://justrecoveryforall.ca/
+
+This is the inaugural post of Hypha’s new blog, Dripline. Through Dripline, we’re hoping to share our experiences and reflections as we learn and build technologies together. It’s also an invitation to our friends and neighbours: [come help us build][] the solidarity economy. Principle 6.
+
+   [come help us build]: https://www.eyemole.io/startacoop.html
+
+We’re excited to hear from you!
+Reach out if there are topics you want to discuss or you think we should cover in future posts.
```
Comment (uditvira) [inline _posts/2021-01-23-how-we-co-operate.md]: I would add this in the blog page footer for all dripline posts. This isn't specific to this post.

File: dripline.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Dripline
+menu_title: Dripline
+order: 2
+---
+{% assign posts = site.posts %}
```
Comment (uditvira) [inline dripline.html:7]: Missing explainer (See google doc)

A dripline signifies the area under the outermost branches of a tree. It’s this [interface](https://permacultureprinciples.com/principles/_11/) between worlds where interesting and productive things happen.

**Review by uditvira — DISMISSED** [review]

**Comment by patcon** [comment]
Thanks for this review comments @uditvira and @dcwalk.

To help not step on @hyphacoop/infrastructure-wg's toes as they troubleshoot, going to make a new PR against `staging` and avoid commits here. No need to transfer your comments into the new one tho

File: _posts/2021-01-23-how-we-co-operate.md (hunk 1 above)
Comment (patcon) [reply _posts/2021-01-23-how-we-co-operate.md]: oooooo every page of website. i was missing that through all these convos. sorry both! cc @dcwalk

**Comment by patcon** [comment]
Expected? https://hyper.distributed.press/hypha.coop/

**Comment by patcon** [comment]
@benhylau is this deploying correctly? https://travis-ci.com/github/hyphacoop/hypha.coop/builds/217779595#L911

Build looks green, but error shows in output, ~~and no update to https://ipfs.distributed.press/ipns/hypha.coop/ .~~ nevermind updated, but still confused about error :)

Also, it looks like dweb only deploys from staging. Is that correct?

~~In the meantime, I've manually deployed to IPFS here: https://ipfs.io/ipfs/QmaZTs7xZRbqnteQhwTECJxRcu5ybG2CzqJDxzbzTXfenc~~
https://ipfs.distributed.press/ipns/hypha.coop

**Comment by benhylau** [comment]
- [x] 1. https://ipfs.distributed.press/ipns/hypha.coop/ looks fine except the missing green dots, you can see the 404s in Inspector, it's likely a missed rel-link (fixed)
- [x] 2. I actually think the site without the green dots is a lot more readable. Maybe I am staring at the screen too much but the green dots make it straining to try to actually read the text @garrying I really hope we can make it not cut into the text, esp for narrow screen (re-ticketed here https://github.com/hyphacoop/hypha.coop/issues/25)
- [x] 3. DWeb temporarily deploys only from staging, but when we merge we should switch it to deploy only on master (see below)
- [ ] 4. See https://staging.compost.digital bottom right corner, how we show links for hyper and ipns, along with their href via the http proxy. I imagine we can do the same here. (the hyper won't work yet but hopefully soon!)

**Comment by patcon** [comment]
Thanks ben!
1. re-ticketed
2. fixed
3. This will need to be applied before merge:
```diff
diff --git a/.travis.yml b/.travis.yml
index 0bfaf54..6a39fe7 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -42,7 +42,7 @@ deploy:
     skip_cleanup: true
     on:
       branch:
-        - staging
+        - master
 
 notifications:
   email: false
```
4. added for IPFS (will leave hyper off until working)

EDIT: Would we like staging.hypha.coop working, or is that just too much effort?

**Comment by patcon** [comment]
@garrying curious your hot-take on moving scaling-x background images of green dots from inline css to external stylesheet, in case I'm breaking some sort of performant thing: e467bfe (did it because our dweb'erizing script search-replaces links into proper format, and only knows how to find css style matches in css files)

**Review by patcon — APPROVED** [review]

**Pull request merged**